src/common/mfu_flist_walk.c

```diff
@@ -50,6 +50,7 @@ static const char** CURRENT_DIRS;
 static flist_t* CURRENT_LIST;
 static int SET_DIR_PERMS;
 static int REMOVE_FILES;
+static mfu_file_t** pfile;
```

Comment:
To make this more obvious that it's a global, let's rename ```pfile``` to something like ```CURRENT_PFILE```.

Reply:
@adammoody ok done